AdyenActions/Components/Redirect/OpenExternalAppDetector.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import UIKit
+
+/// Detects if an external app was opened
+public protocol OpenExternalAppDetector {
```

Comment:
Looks great! I think this could also be `@spi`

Reply:
Actually can be internal 👌